scripts/mojang-block.ts

```diff
@@ -0,0 +1,5001 @@
+/**
+ * All possible MinecraftBlockTypes
+ */
+export const enum MinecraftBlockTypes {
```

Comment:
Unforuntate that Microsoft is now allowing the `vanilla-data` repo for use at the moment. Hopefully that problem is amended, maintaining this file is a problem.